### ISSUE TITLE
Code actions didn't work with nvim-java

### ISSUE THREAD

**Issue by GustavoJCL** [aznhe21/actions-preview.nvim#50]
Hi, i installed `nvim-java` and `actions-preview.nvim`, but I the code actions doesn't work
Neovim version: 0.10
Operating system: Arch Linux
`actions-preview.vim` config:
```
'aznhe21/actions-preview.nvim',
	lazy = true,
	opts = function(_, opts)
		opts.telescope = {
			sorting_strategy = 'ascending',
			layout_strategy = 'vertical',
			layout_config = {
				width = 0.7,
				height = 0.8,
				prompt_position = 'top',
				preview_cutoff = 20,
				preview_height = function(_, _, max_lines)
					return max_lines - 25
				end,
			},
		}
	end,
```
`nvim-java`:
 ```
{
    'nvim-java/nvim-java',
    lazy = true,
    ft = 'java',
    dependencies = {
      'nvim-java/lua-async-await',
      'nvim-java/nvim-java-core',
      'nvim-java/nvim-java-test',
      'nvim-java/nvim-java-dap',
      'MunifTanjim/nui.nvim',
      'neovim/nvim-lspconfig',
      'mfussenegger/nvim-dap',
      'nvim-java/nvim-java-refactor',
      {
        'williamboman/mason.nvim',
        opts = function(_, opts)
          if not opts.registries then
            opts.registries = { 'github:mason-org/mason-registry' }
          end
          table.insert(opts.registries, 1, 'github:nvim-java/mason-registry')
        end,
      },
    },
    config = function()
      require('java').setup()
      require('lspconfig').jdtls.setup {
            settings = {
              java = {
                inlayHints = {
                  parameterNames = {
                    enabled = 'all',
                    exclusions = { 'this' },
                  },
                },
              },
            },
          }
    end,
  }
```

**Comment by aznhe21** [comment]
It seemed to be working, but after updating the package, it stopped functioning... Since `vim.lsp.buf.code_action()` works, it appears to be a bug. I will investigate further.

**Comment by aznhe21** [comment]
Fixed in #52. Please check it.
